masterlist.yaml

```diff
@@ -23681,26 +23681,16 @@ plugins:
     group: *highPriorityGroup
     inc: [ 'Vaults of Skyrim 1.0 NEXUS.esp' ]
     msg:
+      - <<: *compatPatch
+        subs: [ '[Palaces and Castles Enhanced Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/33364/)' ]
       - <<: *patch3rdParty_LotD
         condition: 'active("LegacyoftheDragonborn.esm") and not active("DBM_PCE_Patch.esp") and version("LegacyoftheDragonborn.esm", "5.0.24", >=)'
-      - <<: *patchProvided
-        subs: [ 'Enhanced Lights and FX' ]
-        condition: 'active("EnhancedLightsandFX.esp") and not active("PCE - ELFX PATCH.esp")'
-      - <<: *patchProvided
-        subs: [ 'Immersive Citizens' ]
-        condition: 'active("Immersive Citizens - AI Overhaul.esp") and not active("PCE - Immersive Citizens AI Overhaul Patch.esp")'
-      - <<: *patchProvided
-        subs: [ 'Interesting NPCs' ]
-        condition: 'active("3DNPC.esp") and not active("PCE - 3DNPC Patch.esp")'
       - <<: *patchProvided
         subs: [ 'Open Cities' ]
         condition: 'active("Open Cities Skyrim.esp") and not active("PCE - Open Cities Patch.esp")'
       - <<: *patchProvided
         subs: [ 'Relighting Skyrim' ]
         condition: 'active("RelightingSkyrim_SSE.esp") and not active("PCE - Relighting Skyrim.esp")'
-      - <<: *patchProvided
-        subs: [ 'Skyrim Project Optimization ESM' ]
-        condition: 'not active("PCE - SPO Patch.esp") and active("Skyrim Project Optimization - Full Version.esm")'
       - <<: *patchProvided
         subs: [ 'Skyrim Sewers' ]
         condition: 'active("SkyrimSewers.esp") and not active("PCE - Skyrim Sewers Patch.esp")'
```

Comment:
There appears to be an updated version of this patch in the patch collection too, did you mean to keep this message?

Reply:
oof i missed that my bad